Basic/APIDiorama/Assets/Editor/Tutorials/DiscordIcon.png.meta

```diff
@@ -0,0 +1,123 @@
+fileFormatVersion: 2
```

Comment:
Adding a note here, but this concerns other files:
We should setup git lfs if we are to include binary files like the image above. Else, long term this repo is gonna become a mess.

Reply:
(A slowed down mess of binary diffs being applied everywhere) --> git isn't made for binary assets

Reply:
> We should setup git lfs if we are to include binary files like the image above. Else, long term this repo is gonna become a mess.

I agree (we could also use PlasticSCM since we own it, even though I don't know if it's a viable workflow for our users), but that's an operation that needs to be done at repo level. Maybe we can do it as a separate PR, after doing a pass with the [repo cleaner](http://rtyley.github.io/bfg-repo-cleaner)?

Reply:
Given that not answer was given, I assume this will be addressed in a spearate PR.